cypress/e2e/room_board/elements_in_board_card/createEditDeleteFolderWithFileUploadInRoomBoard.feature

```diff
@@ -21,30 +21,30 @@ Feature: Room Board - Create, edit and delete folder in board, including file ha
         When I click on icon Plus to add content into card
         When I select 'file-folder' from the element selection dialog box
         When I enter name '<folder_name_edited>' for file folder in card
-        When I approve new folder name in card
+        When I click outside of the card to save it
         Then I see a folder with name '<folder_name_edited>' in the card
 
-        # editor removes folder name and name is reset to standard folder name
+        # editor removes folder name and name is reset to previous folder name
         When I click on the page outside of the column
         When I click on the three dot on the card
         When I click on the option Edit in the three dot menu on the card
         When I clear folder name in card
-        When I approve new folder name in card
+        When I click outside of the card to save it
         When I click on the page outside of the column
-        Then I see a folder with name '<standard_folder_name>' in the card
+        Then I see a folder with name '<folder_name_edited>' in the card
```

Comment:
its confusing here, again you edited the name which you already had done in earlier step, also don't know the rational behind re-editing the same name. to me checking whether default naming of folder is missing in the current feature